Python-Home-Challenges/app.py

```diff
@@ -0,0 +1,10 @@
+f = open("Challenge_01.txt", "r")
```

Comment:
It is Better using full file path

Reply:
used `with open("./Python-Home-Challenges/Challenge_01.txt") as file:`